.github/workflows/codespell.yml

```diff
@@ -0,0 +1,23 @@
+name: Codespell
+
+on: ['pull_request']
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - uses: actions/checkout@v3
+      
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v26.1
+        with:
+          files_ignore: pages.*/*/* # Currently ignores all other languages except English
+        
+      - uses: codespell-project/actions-codespell@master
+        with:
+          ignore_words_file: .github/codespell-ignorelist
+          only_warn: 1 # Will exit with 0 regardless
```

Comment:
```suggestion
          # Exit with success regardless of typos.
          only_warn: 1
```

Reply:
```suggestion
          # Exit with 1 regardless of typos.
          only_warn: 1 
```

Reply:
I get it wrong? The CI always pass, just inform the typos, right?

Reply:
Yes, the CI will always pass since the command gives an exit code of 0.

I just edited my suggestion because I got confused. 1→0

Reply:
Yeah, that's it.

Reply:
Are you okay with me applying my suggestion?
I don't mind your one however specifying the exit code is a bit more technical for anyone who happens to read it (just depends on the style how we want the comments)